docs/admin/kubeadm.md

```diff
@@ -108,13 +109,15 @@ By default, `kubeadm init` does not set node CIDR's for pods and allows you to
 bring your own networking configuration through a CNI compatible network
 controller addon such as [Weave Net](https://github.com/weaveworks/weave-kube),
 [Calico](https://github.com/projectcalico/calico-containers/tree/master/docs/cni/kubernetes/manifests/kubeadm)
-or [Canal](https://github.com/tigera/canal/tree/master/k8s-install/kubeadm).
-If you are using a compatible cloud provider or flannel, you can specify a
-subnet to use for each pod on the cluster with the `--pod-network-cidr` flag.
+or [Canal](https://github.com/tigera/canal/tree/master/k8s-install/kubeadm), solutions
+that aren't compatible with this setting.
```

Comment:
I thought it would fail if an user set the flag and tried to run one of those pod networks... Seems that I was wrong :smile:

Ok, I'll update it